### ISSUE TITLE
farmer dashboard 

### ISSUE THREAD

**Issue by elhenriq** [precision-sustainable-ag/On-farm-Protocols#13]
I think that the farmer dashboard should say the correlation between farm code on the field ID and the numerical sequential order, because it would be hard for the grower to know which farm is which on the water tab.

**Comment by brianwdavis** [comment]
Yep, that's a bug that needs to be fixed. I'm trying to move away from
having the three-letter codes be visible at all, and just show the
numerical order.
